### PR TITLE
Allow referral to load even with deleted assignee user

### PR DESCRIPTION
**Pull request by sldblog**
## What does this pull request do?

IPB-207: Let the referral pages load even if the caseworker (assignee) user no longer exists in hmpps-auth (the identity provider).

| Before | After |
| --- | --- |
| image | image |

What is worse, the current response from auth won't be visible to end-users, so they have no hint towards what is wrong.

## What is the intent behind these changes?

Graceful degradation: an external 404 Not Found response should block other users from viewing or progressing with the referral.

### CODE REVIEW

**Comment by kth13** [comment]
@sldblog Is the content team ok with this change?

**Comment by sldblog** [comment]
@kth13 I don't have approval yet.

However, a potentially poorly named "deleted user" is a major win against the current situation (none of the pages load), so waiting for approval will delay people waiting to do work.

**Review by kth13 — APPROVED** [review]
LGTM

**Pull request merged**
